Docs: clarify image store for Kiwi images (#667)
SUSE/doc-susemanager: modules/reference/pages/images/images-stores.adoc

.Image stores for Kiwi build type
[NOTE]
====

> Kiwi build type is used to build system, virtual, and other images. Image store for Kiwi build type is pre-defined as a filesystem directory ([path]``/srv/www/os-images`` on the server) served via HTTP (https://example.com/pub/os-images/) by {productname}. The Image store location is unique and not customizable yet. 

Reviewer: ```suggestion
The Kiwi build type is used to build system, virtual, and other images. 
The image store for the Kiwi build type is pre-defined as a filesystem directory at [path]``/srv/www/os-images`` on the server.
{productname} serves the image store over HTTP from [path]``https://example.com/pub/os-images/``.
The image store location is unique and is not customizable. 
```
